Week2/homework/maartjes-work.js

```diff
@@ -47,15 +47,19 @@ const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const hourlyDuration = tasks.map(elem => elem.duration / 60);
+  const moreTwoHours = hourlyDuration.filter(elem => elem >= 2);
+  const sumHours = moreTwoHours.reduce((pv, cv) => pv + cv);
+  const total = sumHours * hourlyRate;
+  return total;
```

Comment:
👍 for the logic. However, the name of arguments inside the `map`, `filter` and `reduce` functions aren't the best. Maybe you could use `task` or `hours` or `earnings` instead. What do you think?

Reply:
Yes, I also think that words you suggest look more suitable.
 Thank you in advance for your approval and concern.